pkg/stub/config.go

```diff
@@ -282,14 +290,17 @@ func (h *Handler) processError(opcfg *v1.Config, ctype v1.ConfigConditionType, c
 
 		// most reasons for degraded status are handled in ClusterOperatorStatusDegradedCondition based on
 		// state in the opcfg correlated with other factors; but when the error should directly drive the
-		// reason text, we set the reason here; at this time, that means either API Server errors or
-		// errors accessing the local file system
+		// reason text, we set the reason here; at this time, that means either API Server errors,
+		// errors accessing the local file system, or global imagestream config blocking the use of our configured
+		// registry
 		reason := kerrors.ReasonForError(err)
 		switch {
 		case len(reason) > 0 && reason != kapis.StatusReasonUnknown:
 			status.Reason = fmt.Sprintf("APIServer%sError", reason)
 		case cstatus == corev1.ConditionUnknown && ctype == v1.SamplesExist:
 			status.Reason = "FileSystemError"
+		default:
+			status.Reason = "GlobalImageStreamConfigRegistryMismatch"
```

Comment:
A doc update around this new reason for our config invalid condition is probably warranted @dperaza4dustbit

Reply:
got it